test/Npgsql.PluginTests/NodaTimeInfinityTests.cs

```diff
@@ -281,6 +281,86 @@ public async Task DateConvertInfinity()
         }
     }
 
+    [Test]
+    public async Task Interval_write()
```

Comment:
Is there a specific reason these tests can't be done with AssertType and friends?

Reply:
I pretty much just copied tests for `Date` from above. So we probably can do that, but let's make a separate pr targeting all of them together.